raiden/tests/unit/utils/test_filters.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+from eth_utils import to_bytes, to_int
+
+from raiden.tests.utils import factories
+from raiden.utils.filters import (
+    get_filter_args_for_all_events_from_channel,
+    get_filter_args_for_specific_event_from_channel,
+)
+
+
+def test_get_filter_args(contract_manager):
+    channel_identifier = factories.UNIT_CHANNEL_ID
+    token_network_address = factories.UNIT_TOKEN_NETWORK_ADDRESS
+
+    event_filter_params = get_filter_args_for_all_events_from_channel(
+        token_network_address=token_network_address,
+        channel_identifier=channel_identifier,
+        contract_manager=contract_manager,
+    )
+
+    assert event_filter_params["topics"][0] is None
+    assert to_int(hexstr=event_filter_params["topics"][1]) == channel_identifier
+    assert to_bytes(hexstr=event_filter_params["address"]) == token_network_address
+    assert event_filter_params["fromBlock"] == 0
+    assert event_filter_params["toBlock"] == "latest"
+
+    with pytest.raises(ValueError):
```

Comment:
Some short comment on what this tests and why it fails would be nice

Reply:
Done.